### ISSUE TITLE
requirejs can not load sendbird module

### ISSUE THREAD

**Issue by croxt1120** [smilefam/SendBird-SDK-JavaScript#64]
my requirejs config
require.config : {
   sendbird : 'lib/SendBird.min'
}

error : SendBird.min.js:formatted:4933 Uncaught TypeError: Cannot read property 'newInstance' of undefined
    at e (SendBird.min.js:formatted:4933)
    at Object.execCb (require.js:1696)
    at Module.check (require.js:883)
    at Module.enable (require.js:1176)
    at Module.init (require.js:788)
    at callGetModule (require.js:1203)
    at Object.completeLoad (require.js:1590)
    at HTMLScriptElement.onScriptLoad (require.js:1717)
e @ SendBird.min.js:formatted:4933
execCb @ require.js:1696
check @ require.js:883
enable @ require.js:1176
init @ require.js:788
callGetModule @ require.js:1203
completeLoad @ require.js:1590
onScriptLoad @ require.js:1717

**Comment by steveworkman** [comment]
+1 to this, I've got a slightly different error message, but basically it's the same:

```js
core.js?1516032257662-desktop:25 Uncaught TypeError: Cannot read property 'newInstance' of undefined
    at e (SendBird.min.js?yelldotcom:1516032257662-desktop:8)
    at Object.executeDefFunc (core.js?1516032257662-desktop:25)
```

**Comment by steveworkman** [comment]
I believe that the issue is with the UMD implementation, when a browser tries to load this module asynchronously, it can't work out what window is. It's referred to in this bug: https://github.com/umdjs/umd/pull/125

Patches went in to umdjs a few months back. I'd send a PR, but I there's no unminified source here.

**Comment by sf-jay-yun** [comment]
Thanks a lot for this. Will investigate, discuss internally and find what we can support and get back.

**Comment by Alchemist85K** [comment]
@steveworkman @croxt1120 
This issue was resolved at latest JS SDK as version 3.0.55.
Please, install latest SDK and check this issue.
If you still has same issue, tell me in detail about the issue.
Thank you.
